lib/rules/selector-list-comma-newline-before/index.js

```diff
@@ -49,7 +49,7 @@ const rule = function(expectation, options, context) {
           ? ruleNode.raws.selector.raw
           : ruleNode.selector;
         commaIndices
-          .sort()
+          .sort((a, b) => a - b)
```

Comment:
Maybe `.sort((a, b) => b - a)` is a bit better than `.sort((a, b) => a - b).reverse()`